data/json/monsters/bird.json

```diff
@@ -272,5 +272,64 @@
     "type": "MONSTER",
     "copy-from": "mon_duck_chick",
     "upgrades": { "age_grow": 16, "into": "mon_goose_canadian" }
+  },
+    {
+    "id": "mon_crow_mutant_small",
+    "type": "MONSTER",
+    "name": { "str": "oversized crow" },
+    "description": "A horrible bird grown three or four times bigger than ought to be possible and sporting a swollen gut that might look comical if it weren't for the razor sharp beak and the madness in its beady eyes.",
+    "bodytype": "bird",
+    "default_faction": "small_animal",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "BIRD" ],
+    "volume": "3000 ml",
+    "weight": "4000 g",
+    "hp": 8,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "v",
+    "color": "dark_gray",
+    "aggression": 80,
+    "morale": 90,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "melee_cut": 2,
+    "melee_skill": 2,
+    "dodge": 1,
+    "harvest": "bird_large",
+    "fear_triggers": [ "PLAYER_CLOSE" ],
+    "special_attacks": [ [ "SHRIEK", 10 ] ],
+    "death_function": [ "NORMAL" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 8 },
+    "upgrades": { "half_life": 10, "into_group": "GROUP_CROW_MUTANT" },
+    "flags": [ "SEES", "HEARS", "SMELLS", "PATH_AVOID_DANGER_1", "WARM", "FLIES" ]
+  },
+  {
+    "id": "mon_crow_mutant",
+    "type": "MONSTER",
+    "name": { "str": "bloated corvid" },
+    "description": "This may have once been a crow or a raven, but now it's a deformed monstrosity nearly the size of a man. It heaves its sable bulk awkwardly along the ground, too overfed on carrion to fly.",
```

Comment:
```suggestion
    "description": "This may have once been a crow or a raven, but now it's a deformed monstrosity nearly the size of a man.  It heaves its sable bulk awkwardly along the ground, too overfed on carrion to fly.",
```